### DISCUSSION TITLE
Are the accessor attribute, mutator attribute, and weak attribute supported in Corinna?

### DISCUSSION THREAD

**Discussion by yuki-kimoto** [Ovid/Cor#84] [Q&A] [answered]
Hi, I return back to this place because I finished to study Object::Pad.

The accessor attribute, mutator attribute, and weak attribute exist in Object::Pad and P5P MVP

Object::Pad

https://metacpan.org/pod/Object::Pad#field

P5P MVP

https://github.com/Ovid/Cor/blob/master/rfc/mvp.md#1424-4-various-convenience-attributes--

However, the accessor attribute, mutator attribute, and weak attribute don't exist in the "Section 6: Fields"

https://github.com/Ovid/Cor/blob/master/rfc/attributes.md

Are the accessor attribute, mutator attribute, and weak attribute supported in Corinna?

**Comment by Ovid** [comment, marked as answer]
They do not exist in Corinna. Perhaps after the MVP.

*Update*: [I see that @leonerd includes them in Stage 4 of the MVP](https://github.com/Ovid/Cor/blob/master/rfc/mvp.md#1427-7-method-modifiers-around-before-after). Please read that document to see what is planned for the MVP.

**Reply by yuki-kimoto** [reply]
I change this question because I don't yet understand the relations of <code>Corinna</code> and <code>MVP</code> and <code>P5P MVP</code> and <code>MMVP</code>.

Are the <code>accessor</code> attribute, <code>mutator</code> attribute, and <code>weak</code> attribute implemented in <code>use feature 'class'</code>.

**Reply by Ovid** [reply]
The full Corinna MVP spec will not be implemented at this time. The Perl Steering Council (PSC) asked for something smaller than the MVP, so I jokingly referred to it as the "Minimally Minimal Viable Product", or MMVP. That's the RFC that was sent to P5P and that the PSC accepted. This work was planned to be done in seven stages. [If you read this document](https://github.com/Ovid/Cor/blob/master/rfc/mvp.md#1427-7-method-modifiers-around-before-after), you'll see what is intended to be included.

**Reply by yuki-kimoto** [reply]
Sorry, I am so confused.

Isn't Corinna RFC the spec of <code>use feature 'class'</code>?

**Reply by Ovid** [reply]
@yuki-kimoto No. The PSC does not want the full RFC implemented (at first), as explained in [the document I mentioned above](https://github.com/Ovid/Cor/blob/master/rfc/mvp.md).
